src/Packagist/Api/Client.php

```diff
@@ -190,7 +198,7 @@ protected function request($url)
         }
 
         return $this->httpClient
-            ->get($url)
+            ->request('get', $url)
```

Comment:
`get()` is not a method on `ClientInterface`, it's a magic method on `Client`, so I've adjusted this to use the method that belongs to the interface (this is what is called by `Client::get()` by the way)